static/package-lock.json

```diff
@@ -1060,32 +1060,6 @@
       "resolved": "https://registry.npmjs.org/@types/anymatch/-/anymatch-1.3.1.tgz",
       "integrity": "sha512-/+CRPXpBDpo2RK9C68N3b2cOvO0Cf5B9aPijHsoDQTHivnGSObdOF2BRQOYjojWTDy6nQvMjmqRXIxH55VjxxA=="
     },
-    "@types/eslint": {
```

Comment:
just wanted to check we're ok removing these even though all we did package.json wise was install a new package?

Reply:
I ran another fresh install and now less packages are being removed. I think it might have been my node version that caused it to change the package lock so much